packages/jaeger-ui/src/model/trace-dag/DenseTrace.tsx

```diff
@@ -22,6 +22,7 @@ function convSpans(spans: Span[]) {
   const ids: string[] = [];
   spans.forEach(span => {
     const { spanID: id, operationName: operation, process, references, tags: spanTags } = span;
+    const spanGroup = span.group;
```

Comment:
used only once, unnecessary const, makes it harder to grok the actual use of group